mongodbatlas/fw_data_source_mongodbatlas_atlas_user_test.go

```diff
@@ -0,0 +1,127 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"go.mongodb.org/atlas-sdk/v20230201002/admin"
+)
+
+func TestAccConfigDSAtlasUser_ByUserID(t *testing.T) {
+	SkipIfTFAccNotDefined(t)
+	var (
+		dataSourceName = "data.mongodbatlas_atlas_user.test"
+		userID         = os.Getenv("MONGODB_ATLAS_PROJECT_OWNER_ID")
+		user           = fetchUser(userID, t)
+	)
+	resource.Test(t, resource.TestCase{ // does not run in parallel to avoid changes in fetched user during execution
+		PreCheck:                 func() { testAccPreCheckBasic(t); testAccPreCheckBasicOwnerID(t) },
+		ProtoV6ProviderFactories: testAccProviderV6Factories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDSMongoDBAtlasUserByUserID(userID),
+				Check: resource.ComposeTestCheckFunc(
+					dataSourceChecksForUser(dataSourceName, user)...,
+				),
+			},
+		},
+	})
+}
```

Comment:
what about if there are multiple PRs opens? Would these tests fail?

Reply:
This is a good point. 
From CI logs I see this tests takes a total of 2.37s so given the short timeframe I see very little risk of another PR tests modifying this user during that time frame.
We can leave a comment with this concern, and if it becomes a problem we adjust how the test verifies values from the user.

Reply:
My comment is not a blocker but I just want to make sure that there is a risk involved in running this test in the CI. With that said, we can always disable the test in the CI if it gets too flaky. We will keep 👀 on them